### PR TITLE
How your code gets linted

### PR DESCRIPTION
**Pull request by brzpegasus**
~~TODO: Find a better title.~~

### CODE REVIEW

File: source/posts/2016-02-26-how-your-code-gets-linted.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+twitter: "edeblois"
+github: "brzpegasus"
+published: true
+tags: ember, javascript
+summary: "`ember-suave`: Behind the scenes"
+---
+
+Back in August, I wrote a [blog post][suave-up-your-code] introducing [`ember-suave`][ember-suave], an addon that we created at DockYard to help enforce a common code style across all of our projects. With the addon installed, we no longer have to spend precious PR review time commenting on code that doesn't align with our established styleguide. Travis will happily catch those formatting errors by failing the build. Even during development, as files are modified, the linter will re-run and errors will display in the console right away, so you get immediate feedback when you've deviated from the agreed upon style rules.
+
+But how exactly does `ember-suave` lint your files? And when during the build pipeline does the linting happen?
```
Comment (poteto) [inline source/posts/2016-02-26-how-your-code-gets-linted.md]: Maybe `And at what point during the build pipeline does the linting happen?`?

File: source/posts/2016-02-26-how-your-code-gets-linted.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Let's start at the core with JSCS.
```
Comment (poteto) [inline source/posts/2016-02-26-how-your-code-gets-linted.md]: Maybe a little description about what JSCS is for? 

> "JSCS walks through your code and uses predefined rules to automatically lint your code"

File: source/posts/2016-02-26-how-your-code-gets-linted.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Where does `ember-suave` fit in this picture?
+
+If you install `ember-suave`, the Ember CLI addon portion of `broccoli-jscs` isn't used. The goal of `ember-suave` is to augment `broccoli-jscs` by configuring it with a set of rules (both JSCS-built-in rules and custom ones defined inside of `ember-suave`), so you don't have to do so for each project. As such, it has its own `lintTree` implementation that calls out to the `JSCSPlugin`.
+
+## Eager for more?
+
+Even though this post focused on `ember-suave`, you can imagine the process being very similar for other linters, such as JSHint or ESLint.
+
+Understand the inner workings of a build helps tremendously for various situations: when something breaks, you know where and what to look for. Furthermore, you are now better equipped to build extensions of your own, should the need arise.
```
Comment (maabernethy) [inline source/posts/2016-02-26-how-your-code-gets-linted.md]: Understand**ing** the inner workings of a build...

File: source/posts/2016-02-26-how-your-code-gets-linted.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Where does `ember-suave` fit in this picture?
+
+If you install `ember-suave`, the Ember CLI addon portion of `broccoli-jscs` isn't used. The goal of `ember-suave` is to augment `broccoli-jscs` by configuring it with a set of rules (both JSCS-built-in rules and custom ones defined inside of `ember-suave`), so you don't have to do so for each project. As such, it has its own `lintTree` implementation that calls out to the `JSCSPlugin`.
+
+## Eager for more?
+
+Even though this post focused on `ember-suave`, you can imagine the process being very similar for other linters, such as JSHint or ESLint.
```
Comment (rsocci) [inline source/posts/2016-02-26-how-your-code-gets-linted.md]: `focuses` rather than `focused`

**Pull request merged**